Feature/bitmask

japser36 opened the pull request:
Add 16 bit bitmask to pretend to be a 16 bit system

japser36 committed:
Cleanup: pageTable.c formatted

japser36 committed:
Clipped p1 inputs at 16 bits

brisu reviewed (dismissed):
Bitwise overflow to 0 is working as intended. Will edit readme and sample outputs to reflect that change.

brisu committed:
Updated the sample output file with the new overflow bit mask in place

Updated the read me with some more details about the mask and our process that led us there. More info about using integers as binary address representation. Cleaned up the images in the repo to include only in use and relevant info.
